src/praisonai/praisonai/cli/commands/unified.py

```
@@ -0,0 +1,429 @@
+</html>"""
+    
+    index_file = STATIC_DIR / "index.html"
+    index_file.write_text(dashboard_html)
+    return index_file
+
+
+@app.callback(invoke_without_command=True)
+def unified(
+    ctx: typer.Context,
+    port: int = typer.Option(3000, "--port", "-p", help="Port to run unified dashboard on"),
+    host: str = typer.Option("0.0.0.0", "--host", help="Host to bind to"),
```

Ruff (0.15.10)

[error] Possible binding to all interfaces

(S104)